_redirects

```diff
@@ -0,0 +1,2 @@
+/v2/webhooks /v2/webhooks/webhooks-api
```

Comment:
Why this change? All the existing API endpoint documentations have the "API" suffix (which was missing here and I added), but none of the has the page end with -api.

See:
https://developer.dnsimple.com/v2/zones/records/
https://developer.dnsimple.com/v2/domains/

I don't see a reason to rename that page, it will become an exception.

Reply:
File name change reverted in [27346cb](https://github.com/dnsimple/dnsimple-developer/pull/391/commits/27346cba011641569440059f88acb8270d636d87)